views/networking/private_subnet/vms.erb

```diff
@@ -7,12 +11,12 @@
     headers: ["VM", "Private IPv4", "Private IPv6"],
     empty_state: "No VM attached",
     rows:
-      @ps.nics(eager: :vm).map do |nic|
+      @ps.nics(eager: :vm).select { !it.vm_id || displayable_vm_map[it.vm_id] }.map! do |nic|
```

Comment:
can we use `it.vm_id.nil?` instead of `!` because there is already a logical `or` operation and I think putting the `!` instead of `.nil?` reduces readability.

Reply:
Sure, I can make that change.